cmd/root_test.go

```diff
@@ -261,6 +261,50 @@ func TestVerbose(t *testing.T) {
 	}
 }
 
+// TestRoot_effectivePath ensures that the path method returns the effective path
+// to use with the following precidence:  empty by default, then FUNC_PATH
```

Comment:
```suggestion
// to use with the following precedence:  empty by default, then FUNC_PATH
```
It might be useful to search & replace "precidence" for "precedence".